test/algorithm/verifyAuthenticationScheme.test.ts

```diff
@@ -0,0 +1,21 @@
+import { verifyAuthenticationScheme } from "../../src/algorithm/verifyAuthenticationScheme";
+
+describe("verifyAuthenticationScheme()", () => {
+  it("doesn't throw when the authentication scheme is supported", () => {
+    expect(() => {
+      verifyAuthenticationScheme("dpop ");
+      verifyAuthenticationScheme("BeArer ");
+      verifyAuthenticationScheme("DPoP    ");
+      verifyAuthenticationScheme("bearer             ");
+    }).not.toThrow();
+  });
+
+  it("throws when the authentication scheme is not supported", () => {
+    expect(() => {
+      verifyAuthenticationScheme("dpop");
+      verifyAuthenticationScheme("BeArer");
+      verifyAuthenticationScheme("DPoP\t    ");
+      verifyAuthenticationScheme("bearere             ");
+    }).toThrow();
```

Comment:
Throw what?

Reply:
An `AuthenticationSchemeVerificationError` of course!

https://github.com/solid/access-token-verifier/pull/19/commits/bbddfb4c90499468ee4d68d2d7c4c29d97576f72